3ds Max/Max2Babylon/Exporter/BabylonExporter.cs

```diff
@@ -920,34 +920,34 @@ private void moveFileToOutputDirectory(string sourceFilePath, string targetFileP
         private BabylonNode exportNodeRec(IIGameNode maxGameNode, BabylonScene babylonScene, IIGameScene maxGameScene)
         {
             BabylonNode babylonNode = null;
-            try
-            {
-                switch (maxGameNode.IGameObject.IGameType)
-                {
-                    case Autodesk.Max.IGameObject.ObjectTypes.Mesh:
-                        babylonNode = ExportMesh(maxGameScene, maxGameNode, babylonScene);
-                        break;
-                    case Autodesk.Max.IGameObject.ObjectTypes.Camera:
-                        babylonNode = ExportCamera(maxGameScene, maxGameNode, babylonScene);
-                        break;
-                    case Autodesk.Max.IGameObject.ObjectTypes.Light:
-                        babylonNode = ExportLight(maxGameScene, maxGameNode, babylonScene);
-                        break;
-                    case Autodesk.Max.IGameObject.ObjectTypes.Unknown:
-                        // Create a dummy (empty mesh) when type is unknown
-                        // An example of unknown type object is the target of target light or camera
-                        babylonNode = ExportDummy(maxGameScene, maxGameNode, babylonScene);
-                        break;
-                    default:
-                        // The type of node is not exportable (helper, spline, xref...)
-                        break;
-                }
-            }
-            catch (Exception e)
-            {
-                this.RaiseWarning(String.Format("Exception raised during export. Node will be exported as dummy node. \r\nMessage: \r\n{0} \r\n{1}", e.Message, e.InnerException), 2);
-            }
-
+            try
```

Comment:
Can we undo the current branch changes to this chunk? I don't notice a difference other than what's most likely whitespace changes